### ISSUE TITLE
building wine-8.3+ fails with a "A syscall function must use the stdcall convention" in the ntdll.spec, will build with mainline settings

### ISSUE THREAD

**Issue by silverhikari** [Frogging-Family/wine-tkg-git#980]
OS: Manjaro
GCC: 12.2.1
Tested WINE version: wine-8.3, wine-8.4, wine-8.5

error log: https://paste.ee/p/BwG80

short error log:
```
tools/winebuild/winebuild -w --implib -o dlls/ntdll/x86_64-windows/libntdll.a --without-dlltool -b x86_64-w64-mingw32 --export \
  ../wine-git/dlls/ntdll/ntdll.spec
../wine-git/dlls/ntdll/ntdll.spec:1715: A syscall function must use the stdcall convention
make: *** [Makefile:133337: dlls/ntdll/x86_64-windows/libntdll.a] Error 1
make: *** Waiting for unfinished jobs....
==> ERROR: A failure occurred in build().
    Aborting...
  -> Removed BIG_UGLY_FROGMINER - Ribbit
  -> Removed Proton-tkg token - Valve Ribbit
  -> exit cleanup done
```
i was able to get wine-8.2 to build using the default-tkg mode and the higher version such as wine-8.3 to build when using the mainline option

**Comment by silverhikari** [comment]
found it is caused by this commit https://github.com/wine-mirror/wine/commit/921367fc57ccfe9a6a5d3db38f2fc39e46d74e98

**Comment by silverhikari** [comment]
fixed by ff6ff5e652c0bea669fcd988a578a0864dc05dc8
